src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, DATE_FORMAT);
+        LocalDate endDate = LocalDate.parse(dateTo, DATE_FORMAT);
+        StringBuilder result = new StringBuilder("Report for period ");
```

Comment:
```suggestion
        StringBuilder reportBuilder = new StringBuilder("Report for period ");
```
variable names should be informative